en_US/configuration/configuration.md

```diff
@@ -149,6 +149,15 @@ The value of HOCON will be overridden hierarchically, the rules are as follows:
 - In the same file, the value defined in the later section will override any previous key value.
 - A higher-level value will replace that of a lower-level.
 
+the priority of configuration is Environment Variables, `etc/emqx.conf` file and HTTP API.
+This means that for a newly started EMQX, the configuration settings in environment variables and
+`etc/emqx.conf` will always override any configurations made in HTTP API.
+
+If the configuration is modified through the HTTP API while EMQX is running, it will take effect immediately in the cluster. However, if EMQX is restarted, these modified configurations may be overwritten by the Environment Variables or etc/emqx.conf
+
+Configurations modified via the HTTP API will take effect during EMQX running. However, after
+restarting EMQX, they may be overwritten by higher priority Environment Variables or `etc/emqx.conf`, resulting in lost these configurations. Therefore, the best approach is to manually persist all configurations modified via HTTP API to the `etc/emqx.conf` file to prevent them lost.
```

Comment:
The order of priority for configuration is:
environment variables, 'etc/emqx.conf', and 'data/configs/cluster.hocon'.
1. Settings in environment variables starting with 'EMQX_' are given the highest priority.
2. Configuration settings in the 'etc/emqx.conf' file may be overwritten by environment variables.
3. Changes made through the EMQX dashboard UI, management HTTP API, or CLI
   overwrite the 'data/configs/cluster.hocon' file at runtime.
 4. If you modify a configuration setting using the API, the change takes effect immediately.
 5. However, if the same setting is configured differently in the 'etc/emqx.conf' file,
   'etc/emqx.conf' takes priority after a reboot.

Reply:
which one is better?